wrangler-service/src/main/java/co/cask/wrangler/service/spanner/SpannerSpecification.java

```diff
@@ -33,6 +34,7 @@
 
   SpannerSpecification(String serviceFilePath, String project, String instance,
                        String database, String table, Schema schema) {
+    this.referenceName = "Spanner_Source";
```

Comment:
this is a bad reference name as it would be the same for any spanner connection created by dataprep. Is this what the other connections do?

A better one would be something like '<database>:<table>', though you would have to make sure whatever special characters CDAP doesn't allow are replaced.

Reply:
@albertshau comment addressed.